application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -149,6 +149,8 @@ analytics.livetable.startDate=Start Date
 analytics.livedata.action.rowEvolution.title=Row Evolution
 
 ## Others
+
+rendering.macro.accessDenied=Please contact the administrator to resolve this issue.
```

Comment:
``rendering.macro.`` prefix should be followed by a macro name. There's no ``accessDenied`` macro AFAIK.

```suggestion
analytics.accessDenied=You are not allowed to view analytics information.
```